### DISCUSSION TITLE
How to install package for prod?

### DISCUSSION THREAD

**Discussion by sathiasri** [IlyaMatsuev/Apex-GraphQL-Client#28] [Q&A]
Hi,
 I can only find link to sandbox and devour to install the package. How do I do it for production orG?

**Comment by IlyaMatsuev** [comment]
Hi,
Dev orgs share the same login URL as the production orgs. You should be able to use the same link:  
http://login.salesforce.com/packaging/installPackage.apexp?p0=04t5Y000001zNZLQA2

Please let me know if that works
